src/Mod/Part/Gui/Utils.cpp

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************
```

Comment:
Missing SPDX header, and this is the old license block, and is attributed to the wrong person. https://freecad.github.io/DevelopersHandbook/codeformatting/#c-copyright-header